src/Documentation/sidebar.json

```diff
@@ -139,6 +139,10 @@
         "label": "Running DVC on Windows",
         "slug": "running-dvc-on-windows"
       },
+      {
+        "label": "Using Docker",
```

Comment:
let's make it part of the Install section

Reply:
All right)

Reply:
I see both static/docs/install/docker.md and static/docs/user-guide/using-docker.md present in this PR. Should only review docs/install then?